service/src/main/java/uk/gov/mca/beacons/api/legacybeacon/domain/LegacyBeacon.java

```diff
@@ -74,6 +75,10 @@ public class LegacyBeacon extends BaseAggregateRoot<LegacyBeaconId> {
   @NotNull
   private OffsetDateTime lastModifiedDate;
 
+  public String getBeaconStatus() {
```

Comment:
You shouldn't need this new method. Looking at the property there already is a getter, but it's restricted for some reason?

```
@getter(AccessLevel.NONE)
private String beaconStatus;

```